vsintegration/src/vs/FsPkgs/FSharp.VS.FSI/fsiSessionToolWindow.fs

```diff
@@ -587,13 +595,16 @@ type internal FsiToolWindow() as this =
                  // REVIEW: log error into Trace.
                  // Example errors include no active document.
 
-    let onMLSend (sender:obj) (e:EventArgs) =       
-        sendSelectionToFSI false
+    let onMLSendSelection (sender:obj) (e:EventArgs) =       
```

Comment:
when a param is unused, with underscore is better, easier to know is not a bug ( `_sender` `_e` )
I was pretty sure every project had unused vars as error enabled, i'll check